tasmota/xnrg_12_solaxX1.ino

```diff
@@ -155,10 +155,18 @@ void solaxX1_RS485Send(uint16_t msgLen)
     solaxX1Serial->read();
   }
 
+  if (PinUsed(GPIO_SOLAXX1_RTS)) {
+    digitalWrite(Pin(GPIO_SOLAXX1_RTS), HIGH);
+  }
   solaxX1Serial->flush();
   solaxX1Serial->write(message, msgLen);
   solaxX1Serial->write(highByte(crc));
   solaxX1Serial->write(lowByte(crc));
+  solaxX1Serial->flush();
+  if (PinUsed(GPIO_SOLAXX1_RTS)) {

```

Comment:
Flush doesn't do anything here. It's a receive buffer flush. NOT a send buffer flush (Changed by Core team since a year I must confess)